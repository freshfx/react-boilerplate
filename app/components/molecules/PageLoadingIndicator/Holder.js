import styled from '@emotion/styled'

const Holder = styled.div`
  display: flex;
  justify-content: center;
  padding: 5rem;
`

export default Holder
