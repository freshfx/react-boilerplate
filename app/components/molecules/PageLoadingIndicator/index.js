import React from 'react'

import LoadingIndicator from 'components/atoms/LoadingIndicator'

import Holder from './Holder'

const PageLoadingIndicator = () => (
  <Holder>
    <LoadingIndicator />
  </Holder>
)

export default PageLoadingIndicator
