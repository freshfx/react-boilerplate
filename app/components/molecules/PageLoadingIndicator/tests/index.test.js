import React from 'react'
import {render} from '@testing-library/react'

import PageLoadingIndicator from '../index'

describe('PageLoadingIndicator', () => {
  it('should match the snapshot', () => {
    const {container} = render(<PageLoadingIndicator />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
