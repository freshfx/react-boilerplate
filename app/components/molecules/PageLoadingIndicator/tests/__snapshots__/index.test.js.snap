// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoadingIndicator should match the snapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  padding: 5rem;
}

<div
  class="emotion-0 emotion-1"
>
  <span>
    Loading
  </span>
</div>
`;
